pkg/registry/apis/datasource/standalone_services.go

```diff
@@ -73,11 +76,23 @@ func apiBuilderServices(cfg *setting.Cfg, pluginID string) (
 
 	kvStore := kvstore.ProvideService(sqlStore)
 	featureToggles := featuremgmt.ProvideToggles(featureManager)
-	acimplService, err := acimpl.ProvideService(cfg, sqlStore, routeRegisterImpl, cacheService, accessControl, featureToggles)
+	bundleRegistry := bundleregistry.ProvideService()
+
+	quota := quotaimpl.ProvideService(sqlStore, cfg)
+	orgService, err := orgimpl.ProvideService(sqlStore, cfg, quota)
+	if err != nil {
+		return nil, nil, nil, nil, err
+	}
+	teamService := teamimpl.ProvideService(sqlStore, cfg)
+	userService, err := userimpl.ProvideService(sqlStore, orgService, cfg, teamService, cacheService, quota, bundleRegistry)
+	if err != nil {
+		return nil, nil, nil, nil, err
+	}
+
+	acimplService, err := acimpl.ProvideService(cfg, sqlStore, routeRegisterImpl, cacheService, accessControl, userService, featureToggles)
```

Comment:
I did not expect that we had places in the code where wire isn't used

Reply:
yeah, looks a bit strange